### ISSUE TITLE
Unstyled home page after install, with backtrace

### ISSUE THREAD

**Issue by RichVel** [geerlingguy/drupal-vm#1114]
### Issue Type

 - Bug Report / Support Request

### Your Environment

```
Vagrant 1.8.7
VirtualBox 5.1.10r112026
ansible 2.2.0.0
  config file = 
  configured module search path = Default w/o overrides
```
vagrant hostmanager plugin installed and /etc/hosts updated OK.

### Your OS

 - **macOS** 10.11.6
 
### Full console output

[Gist with details](https://gist.github.com/RichVel/03deeefcd7f103451442d8fdb414946c):

- page source
- VM details
- backtraces x 2 from watchdog table - these show a mkdir() failing

### Summary

Did `vagrant up` with this config.yml and Ansible completes fine.  I can see a basic unstyled Drupal home page, and can login, but there are no style sheets.

From the backtraces, it seems like a permissions issue.  

Ansible is logging in as vagrant, and the sudo's work fine.
```
$ ansible -i drupal, drupal -a "id"
drupal | SUCCESS | rc=0 >>
uid=900(vagrant) gid=900(vagrant) groups=900(vagrant),27(sudo),1000(admin)
```

I am using vboxsf shared folders, which I use with other Vagrant VMs all the time.

**Comment by geerlingguy** [comment]
I think the problem could be your Vagrant version—could you either run Vagrant 1.8.6 or Vagrant 1.9.0/1? 1.8.7 had a variety of annoying issues, and this might be a symptom of one of those issues.

**Comment by geerlingguy** [comment]
Also:

> I am using vboxsf shared folders, which I use with other Vagrant VMs all the time.

Is it possible for you to use NFS instead (Drupal VM's defaults), and see if that works. The vboxsf synced dirs _shouldn't_ be an issue, but I wonder if they are.

**Comment by RichVel** [comment]
I've tried a few things that have not helped:

1. Upgraded Vagrant to 1.9.1
2. In VM: `usermod www-data -a - G vagrant`
3. On host: `vagrant vbguest` to ensure Guest Additions match VB 5.1.10 (they were at 5.0.24).

Vbguest now reports:

    [drupalvm] GuestAdditions 5.1.10 running --- OK.

I will try rsync shared folders, then after that NFS (hard to get working on this Mac right now for some reason, though I used NFS for Vagrant all the time only a few months back)

**Comment by geerlingguy** [comment]
@RichVel - Can you also do a `vagrant box update` to try to get the latest version of the box (you'll need to do a `vagrant destroy -f` to remove the existing install too).

**Comment by RichVel** [comment]
I did the box update, but that didn't help.

Changing the config to rsync has worked, so at least I see the home page with styling.  Playbook completes as before.  Config.yml for the record is:
```
drupal_major_version: 7

vagrant_synced_folder_default_type: ''

vagrant_synced_folders:
  - local_path: .
    destination: /var/www/drupalvm
    type: rsync
    create: true
    excluded_paths:
      - drupal/private
      - drupal/public/.git
      - drupal/public/sites/default/files
      - drupal/tmp
#     options_override:
#       owner: "vagrant",
#       group: "www-data",
#       mount_options: ["dmode=775,fmode=664"]
```

All now working well, just logged in with the admin user, admin password.

**Comment by geerlingguy** [comment]
Aha! Someone else had the same issue, and I think it comes down to whether you upgraded an older VM install to the latest Drupal VM version.

Can you run `apache2 -version` and see what version is running?

You might need to run a `sudo apt-get upgrade apache2 -y`, then afterwards if it still is happening, run `sudo service apache2 restart` and `sudo service php[version]-fpm restart`.

**Comment by dzymail** [comment]
i had same issue. 
nfs type not working so i using native shared folder. 
i am not using ubuntu, because my product server using Centos 7.
vagrant 1.9.0 + macOS 10.12 + ansible 2.2

**Comment by rdonkin** [comment]
@geerlingguy - here's the Apache version: 

```
$ apache2 -version
Server version: Apache/2.4.18 (Ubuntu)
Server built:   2016-07-14T12:32:26
```

@dzymail have a look at https://github.com/mitchellh/vagrant/issues/7646, last few messages about clearing out `/etc/exports` on host may help.

**Comment by geerlingguy** [comment]
It looks like the main issues that can cause this include:

  - Older versions of Vagrant, VirtualBox and/or Ansible (make sure you're up to date)
  - Upgrading an older Drupal VM box to a new Drupal VM version without a `vagrant destroy -f && vagrant up` to rebuild
  - An older `local.config.yml` or `config.yml` that doesn't include the updated `apache_extra_parameters` that's in `default.config.yml` (that would definitely be a culprit) for each Drupal vhost.
  - Having the wrong Apache version installed inside the VM (log in and make sure you have Apache 2.4.10+ with `apache2 -version` on Ubuntu/Debian).

Closing out this issue for now, as one of the above should be a fix (at least in the few times I saw people having these issues, that was the fix).
